src/main/java/edu/arizona/kra/proposaldevelopment/PropDevRoutingStateConstants.java

```diff
@@ -0,0 +1,141 @@
+    public static final String ROUTE_STOP_DATE_CRITERIA = " AND actions.nodeStopDate ";
+    public static final String LEAD_UNIT_ANNOT_CRITERIA = " AND actn.ACTN_RQST_ANNOTN_TXT LIKE '";
+    public static final String PROPOSAL_PERSON_NAME_CRITERIA = " AND prop.proposal_number in (select prop.proposal_number from eps_proposal prop,eps_prop_person pers WHERE prop.proposal_number=pers.proposal_number AND lower(pers.FULL_NAME) LIKE '%";  
+    public static final String LEAD_COLLEGE_CRITERIA = " AND prop.owned_by_unit IN (SELECT unit_number FROM unit u START WITH u.UNIT_NUMBER='";
+    public static final String LEAD_COLLEGE_CRITERIA_CONT = "' CONNECT BY PRIOR u.UNIT_NUMBER = u.PARENT_UNIT_NUMBER)";
+    public static final String ORDER_CRITERIA = " ORDER BY stop_date";
+        
+    
+    public static final String DATE_QUERY_PREFIX = "to_date('";
+    public static final String DATE_FORMAT_STR = "','MM/DD/RRRR')";
+    public static final String END_DATE_FORMAT_STR = "','HH24:MI MM/DD/RRRR')";
+    public static final String END_DATE_TIME ="23:59 ";
+    
+    public static final String NODE_NAME_ADHOC = "Ad Hoc";
+    public static final String NODE_NAME_ALL = "All";
+    public static final String NODE_NAME_SPS_APPROVE = "SPS Approve";
+    public static final String NODE_NAME_PI = "PI/Co-PI";
+    public static final String NODE_NAME_LEAD_UNIT = "Lead Unit";
+    public static final String NODE_NAME_COLLEGE = "Department/College";
+    public static final String NODE_NAME_INITIATED = "Returned to Initiator";
+    
+    public static final String NODE_KEY_SPS_APPROVE = "OSPOfficeRouting";
+    public static final String NODE_KEY_PI = "ProposalPersons";
+    public static final String NODE_KEY_LEAD_UNIT = "LeadUnitPreApproval";
+    public static final String NODE_KEY_HIERARCHY = "Hierarchy Request";
+    public static final String NODE_KEY_INITIATED = "Initiated";
+    
+    public static final String YES = "Y";
+    public static final String NO = "N";
+    
+    public static final Integer annotationCriteriaOffset;
+    public static final Map<String, String> SEARCH_QUERIES;
+    public static final Map<String, String> SEARCH_QUERIES_LIKE;
+    static {
+        Map<String, String> aMap = new HashMap<String, String>( );
+        aMap.put(ROUTE_STOP_NAME, NODE_NAME_CRITERIA);
+        aMap.put(PROPOSAL_NUMBER, PROP_NUMBER_CRITERIA);
+        aMap.put(DOCUMENT_NUMBER, DOC_NUMBER_CRITERIA);
+        aMap.put(SPONSOR_CODE, SPONSOR_CODE_CRITERIA);
+        aMap.put(LEAD_UNIT, UNIT_CRITERIA);
+        SEARCH_QUERIES = Collections.unmodifiableMap(aMap);
+        
+        Map<String, String> bMap = new HashMap<String, String>( );
+        bMap.put(PROPOSAL_TITLE, PROP_TITLE_CRITERIA);
+        bMap.put(SPONSOR_NAME, SPONSOR_NAME_CRITERIA);
+        SEARCH_QUERIES_LIKE =  Collections.unmodifiableMap(bMap);
+        
+        annotationCriteriaOffset = SQL_LOOKUP.lastIndexOf("actn.actn_rqst_cd='A'")+21 ;
```

Comment:
This looks magical, it might warrant adding a comment in order to demystify for non-wizards.